File: RADWIMPS.cpp
#include <iostream>

int main() {
  struct RADWIMPS {
    RADWIMPS const &then() const {
      std::cout << "前";
      return *this;
    }

    void 世() const { std::cout << "世\n"; }
  } const radwimps;

  radwimps.then().then().then().世();
}

File: RADWIMPS.java
class RADWIMPS {
    public static void main(String[] args) {
        RADWIMPS RADWIMPS = new RADWIMPS();
        
        RADWIMPS.then().then().then().世();
    }
    
    public void 世() {
        System.out.println("世");
    }
    
    public RADWIMPS then() {
        System.out.print("前");
        return this;
    }
}
